formulas/wick-base/functions/wick-service-add-dependency-systemd

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Internal: Handles adding a dependency specifically if we are
+# running systemd.
+#
+# $1 - The name of the service you wish to add a dependency to.
+# $2 - The name of the service you wish to add as a dependency to the service.
+#
+# Returns nothing.
+wickServiceAddDependencySystemd() {
+    local dependency overridePath paths service
+    wickGetArgument service 0 "$@"
+    wickGetArgument dependency 1 "$@"
+
+    wickServiceOverridePathSystemd paths "$service"
+    wickArrayJoin overridePath "/" "${paths[@]}"
+
+    wickDebug "Adding \"$dependency\" as a dependency of \"$service\""
+    # Add a Wants= after the [Unit] section
```

Comment:
Newline above comment.

Also, this line is triggering the variable matching pattern because it looks like this:

```
# some comment
variable=some_value
```

To fix that you should put a colon in the comment like so

```
#: Add a Wants= after the [Unit] section
```

Or else you can change the sed with some clever string stuff.  I inserted two apostrophes before the equals.

    sed -i 's/^\[\Unit\]/&\nWants''='"$dependency"'/' "$overridePath"

Looks like this is a bug in `tomdoc.sh` - https://github.com/tests-always-included/tomdoc.sh/blob/master/tomdoc.sh#L179 - I will file a bug or fix.